tests/all/js_globals/String.rs

```diff
@@ -0,0 +1,32 @@
+#![allow(non_snake_case)]
+
+use project;
+
+#[test]
+fn char_at() {
+    project()
+        .file("src/lib.rs", r#"
+            #![feature(proc_macro, wasm_custom_section)]
+
+            extern crate wasm_bindgen;
+            use wasm_bindgen::prelude::*;
+            use wasm_bindgen::js;
+
+            #[wasm_bindgen]
+            pub fn string_char_at(string: &js::String, index: i32) -> js::String {
+                string.char_at(index)
+            }
+        "#)
+        .file("test.ts", r#"
+            import * as assert from "assert";
+            import * as wasm from "./out";
+
+            var anyString = 'Brave new world';
```

Comment:
This is taken from the documentation. I haven't added a test case for default `index = 0` as I am not sure currently it's possible